axum/src/docs/extract.md

```diff
@@ -499,6 +499,67 @@ let app = Router::new().route("/foo", get(handler));
 # };
 ```
 
+## Cannot implement both `FromRequest` and `FromRequestParts`
+
+Note that you cannot implement both `FromRequest` and `FromRequestParts` for the
+same type, unless it is wrapping another extractor:
```

Comment:
We should probably note again here the blanket impl, plus make it clear that there's not going to be a compile error from the two direct impls.

```suggestion
Note that you will make your extractor unusable by implementing both `FromRequest` and `FromRequestParts` directly for the
same type, unless it is wrapping another extractor:
```